src/js/views/transfer.js

```diff
@@ -394,12 +403,28 @@ class Transfer extends React.Component {
     });
     this.setState({ transfers: newTransfers });
   }
+    
+  getSpentInputs() {
+    const { inputs } = this.state;
+    return inputs.filter(i => (i.spent && i.selected));
+  }
+    
+  inputNotTransfer(input) {
+    const { transfers } = this.state;
+        
+    for(const k in transfers) {
+      if(transfers[k].address.substring(0, 81) == input.address.substring(0, 81))
+        return false;
+    }
+        
+    return true;
+  }
 
   hasEnoughInputs() {
     const { transfers, donation, inputs } = this.state;
     const totalValue =
       donation.value + transfers.reduce((s, t) => s + t.value, 0);
-    const unspentInputs = inputs.filter(i => (!i.spent || i.selected));
+    const unspentInputs = inputs.filter(i => (!i.spent || i.selected) && this.inputNotTransfer(i));
```

Comment:
Probably `unspetInputs` is no longer the best name, maybe something like `validInputs`